client/src/App.jsx

```diff
@@ -1,18 +1,20 @@
 import React from "react";
-import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
-import Home from "./pages/home.page";
+import { Footer, Projects, Skills, Home, AboutMe, Contacts } from './containers'
+import { Navbar,Quote } from './components'
 
 
 
 const App = () => {
   return (
-    <Router>
-      <Routes>
-        <Route path="/" element={<Home />}>
-          <Route path="/editor" element={<h1>Hello</h1>} />
-        </Route>
-      </Routes>
-    </Router>
```

Comment:
Revert this back and use react outlate which is already and when navbar is rendered then automatically render the **HomePage**